### ISSUE TITLE
Please update activesupport version for cocoapods v1.1.0-beta

### ISSUE THREAD

**Issue by erickyim** [CocoaPods/cocoapods-acknowledgements#57]
Error when using `cocoapods v1.10.0-beta1`

```sh
Error loading the plugin `cocoapods-acknowledgements-1.2.0`.

Gem::ConflictError - Unable to activate cocoapods-acknowledgements-1.2.0, because activesupport-5.2.4.3 conflicts with activesupport (>= 4.0.2, < 5)
```
Consider to release a beta version and  update activesupport dependency since `cocoapods v1.1.0-beta1` requires `activesupport > 5`.

Thanks.

**Comment by dnkoutso** [comment]
yeap we should fix that!

**Comment by dnkoutso** [comment]
Fixed with #58.
